specification/databricks/resource-manager/readme.md

```diff
@@ -18,6 +18,16 @@ batch:
   - tag: package-2023-05-01
 ```
 
+
+### Tag: package-2024-05
+
+These settings apply only when `--tag=package-2024-05` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-05'
+input-file:
+  - Microsoft.Databricks/stable/2024-05-01/databricks.json
+  - Microsoft.Databricks/stable/2024-05-01/vnetpeering.json
+```
```

Comment:
Curie can you look at readme file changes from PR I shared and make it similarly